server/src/main/java/org/opensearch/index/IndexingPressureService.java

```diff
@@ -25,16 +27,18 @@ public IndexingPressureService(Settings settings, ClusterService clusterService)
         shardIndexingPressure = new ShardIndexingPressure(settings, clusterService);
     }
 
-    public Releasable markCoordinatingOperationStarted(long bytes, boolean forceExecution) {
+    public Releasable markCoordinatingOperationStarted(BulkRequest bulkRequest, boolean forceExecution) {
```

Comment:
I see this is only used by TransportBulkAction. Is that the only place this is ever intended to be used? Shouldn't we just move this method out of the service if that's the case? If not then lets refactor the method to take the interface?

```suggestion
    public Releasable markCoordinatingOperationStarted(Accountable accountable, boolean forceExecution) {
```

Reply:
Yes, it makes perfect sense to use an Interface instead and keep the contract clean. Refactored to use a Supplier.